src/Kount/Ris/Request.php

```diff
@@ -592,6 +667,149 @@ public function setBillMeLaterPayment($blmlId) {
   }
 
   /**
+   * Set a apple pay payment type
+   *
+   * @param string $appleId apple pay id
+   * @return this
+   */
+  public function setApplePayment($appleId) {
+    $this->data['PTYP'] = self::APAY_TYPE;
+    return $this->setPaymentToken($appleId);
+  }
+
+  /**
+   * Set a BPAY payment type
+   *
+   * @param string $bppId BPAY id
+   * @return this
+   */
+  public function setBpayPayment($bppId) {
+    $this->data['PTYP'] = self::BPAY_TYPE;
+    return $this->setPaymentToken($bppId);
+  }
+
+  /**
+   * Set a Carte Bleue payment type
+   *
+   * @param string $cbpId Carte Bleue id
+   * @return this
+   */
+  public function setCarteBleuePayment($cbpId) {
+    $this->data['PTYP'] = self::CARTE_BLEUE_TYPE;
+    return $this->setPaymentToken($cbpId);
+  }
+
+  /**
+   * Set a ELV payment type
+   *
+   * @param string $elvpId ELV id
+   * @return this
+   */
+  public function setElvPayment($elvpId) {
+    $this->data['PTYP'] = self::ELV_TYPE;
+    return $this->setPaymentToken($elvpId);
+  }
+
+  /**
+   * Set a GiroPay payment type
+   *
+   * @param string $giroPayId GiroPay id
+   * @return this
+   */
+  public function setGiroPayPayment($giroPayId) {
+    $this->data['PTYP'] = self::GIROPAY_TYPE;
+    return $this->setPaymentToken($giroPayId);
+  }
+
+  /**
+   * Set a Interac payment type
+   *
+   * @param string $interacId Interac id
+   * @return this
+   */
+  public function setInteracPayment($interacId) {
+    $this->data['PTYP'] = self::INTERAC_TYPE;
+    return $this->setPaymentToken($interacId);
+  }
+
+  /**
+   * Set a Mercado Pago payment type
+   *
+   * @param string $mercadoPagoId Mercado Pago id
+   * @return this
+   */
+  public function setMercadoPagoPayment($mercadoPagoId) {
+    $this->data['PTYP'] = self::MERCADO_PAGO_TYPE;
```

Comment:
This one needs to be checked on Kount side -- correct is `MERCADO_PAGO`, but Kount RESTful API says that the parameter value is `MERCADE_PAGO`.
Input from @gallilama required.